integrations/rabbitmq/documentation.yaml

```diff
@@ -0,0 +1,94 @@
+exporter_type: included
+app_name_short: RabbitMQ
+app_name: {{app_name_short}}
+app_site_name: RabbitMQ
+app_site_url: https://rabbitmq.com/
+exporter_name: the RabbitMQ exporter
+exporter_pkg_name: rabbitmq
+exporter_repo_url: https://rabbitmq.com/prometheus.html#overview-prometheus
+additional_prereq_info: |
+  {{app_name_short}} exposes Prometheus-format metrics automatically; you do not
+  have to install it separately. To verify that {{exporter_name}} is emitting metrics on the expected
+  endpoints, set up port-forwarding with the following command:
+
+  To determine if {{exporter_name}} is exposing metrics, run the following command:
+
+  <pre class="devsite-click-to-copy">
+  kubectl -n {{namespace_name}} port-forward {{pod_name}} 15692
+  </pre>
+
+  Access the endpoint `localhost:15692/metrics` by using the browser or curl in another terminal session
+  to verify that the metrics are being exposed by the exporter for scraping.
+dashboard_available: true
+multiple_dashboards: false
+dashboard_display_name: {{app_name_short}} Prometheus Overview
+podmonitoring_config: |
+  apiVersion: monitoring.googleapis.com/v1
+  kind: PodMonitoring
+  metadata:
+    name: rabbitmq
+    labels:
+      app.kubernetes.io/name: rabbitmq
+      app.kubernetes.io/part-of: google-cloud-managed-prometheus
+  spec:
+    endpoints:
+    - port: prometheus
+      scheme: http
+      interval: 30s
+      path: /metrics
+    selector:
+      matchLabels:
+        app.kubernetes.io/component: rabbitmq
+additional_podmonitoring_info: |
+  Ensure that the values of the port and matchLabels match that of the {{app_name_short}} pods you want to monitor.
+  Rabbitmq deployed by the [operator](https://www.rabbitmq.com/kubernetes/operator/operator-overview.html){:class=external}
+  will contain the label `app.kubernetes.io/component: rabbitmq` and `prometheus` port.
```

Comment:
This is great -- I tested these docs by installing RabbitMQ a different way (via [GCP click-to-deploy](https://github.com/GoogleCloudPlatform/click-to-deploy/tree/master/k8s/rabbitmq)) and it had [a different label](https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/k8s/rabbitmq/chart/rabbitmq/templates/statefulset.yaml#L7), but this was good enough to guide me to adapt the config accordingly.